labonneboite/common/search.py

```diff
@@ -372,6 +374,15 @@ def build_json_body_elastic_search(
         }
     })
 
+
+    if departments:
+        regex = '|'.join(departments)
+        filters.append({
+            'regexp': {
+                'department': regex
+            }
+        })
+
```

Comment:
ok tu veux un filtre `office.departement is in departements`

on a déjà un cas similaire `office.naf is in nafs` ci-dessous, pourquoi ne pas réutiliser le même, en évitant la regexp?

![image](https://user-images.githubusercontent.com/10533583/34100431-ac7830a0-e3e2-11e7-94d1-d7ac7ff5e6d7.png)

Reply:
J'avais oublié ce fonctionnement, d'où la regexp ^^